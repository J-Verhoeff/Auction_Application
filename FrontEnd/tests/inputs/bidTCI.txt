login
admin
create
admin1
admin
100
create
user2
sell-standard
10.10
logout
login
admin1
bid
itemNA
admin
bid
item
userNA
bid
item
admin
10
bid
item
admin
200
addcredit
1000
admin1
bid
item
admin
1000
bid
item
admin
20.10
logout
login
user
addcredit
22
bid
item
admin
20.20
bid
item
admin
22
logout
login
user2
bid
logout
exit