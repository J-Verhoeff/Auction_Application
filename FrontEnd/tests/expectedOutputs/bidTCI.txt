
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: User: 
Welcome admin!
User Type: AA
Available Credit: 100.00$

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: Type of user (admin, full-standard, buy-standard, sell-standard): Credit amount: New user successfully created!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: New username: Type of user (admin, full-standard, buy-standard, sell-standard): Credit amount: New user successfully created!

Options:
	logout
	create
	delete
	advertise
	bid
	refund
	addcredit
Transaction: 
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: User: User not in database!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: User: 
Welcome user!
User Type: FS
Available Credit: 0.00$

Options:
	logout
	advertise
	bid
	addcredit
Transaction: Amount of credit: Successfully added credit!
Current credit amount: 22.00 for user

Options:
	logout
	advertise
	bid
	addcredit
Transaction: Item name: Seller's username: No match found for item and user!

Options:
	logout
	advertise
	bid
	addcredit
Transaction: Transaction not in options!


Options:
	logout
	advertise
	bid
	addcredit
Transaction: Item name: Seller's username: No match found for item and user!

Options:
	logout
	advertise
	bid
	addcredit
Transaction: Transaction not in options!


Options:
	logout
	advertise
	bid
	addcredit
Transaction: 
Welcome to the Auction App!
Start entering transactions
To exit the application type 'exit'

Options:
	login
Transaction: User: User not in database!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Transaction not in options!


Options:
	login
Transaction: Goodbye!
